common/Server/eos/core/eos_core.sqf

```diff
@@ -96,7 +104,11 @@ if (_debug) then {diag_log (format ["Spawned Patrol: %1",_counter]);0= [_mkr,_co
 						};
 
 							0=[(_cGroup select 2),"LIGskill"] call eos_fnc_grouphandlers;
-								0 = [(_cGroup select 2),_mkr] call EOS_fnc_taskpatrol;
+							if(isClass(configfile >> "lambs_main_Display")) then {
```

Comment:
We should probably do the same format as detecting ace though the mod path config just encase lambs changes